random-beans/src/test/java/io/github/benas/randombeans/randomizers/range/DateRangeRandomizerTest.java

```diff
@@ -95,4 +100,39 @@ public void whenSpecifiedMaxDateIsNull_thenShouldUseDefaultMaxValue() {
         assertThat(randomDate).isAfterOrEqualsTo(minDate);
     }
 
+    public static class TestData {
+        public TestData(Date date) {
+            this.date = date;
+        }
+
+        @Randomizer(value=DateRangeRandomizer.class, args={
+                // 2016-01-10
+                @RandomizerArgument(value="1452384000000", type=Date.class),
+                // 2016-01-30
+                @RandomizerArgument(value="1454112000000", type=Date.class)
+        })
+        private Date date;
+
+        public Date getDate() {
+            return date;
+        }
+
+        public void setDate(Date date) {
+            this.date = date;
+        }
+    }
+
+    @Test
+    public void annotationShouldWorkWithRange() {
+        DateRangeRandomizerTest.TestData data = null;
+        for(int x=0; x < 10; x++) {
+            data = new EnhancedRandomBuilder().build().nextObject(DateRangeRandomizerTest.TestData.class);
```

Comment:
You can use the shortcut static method `EnhancedRandom.random` here (since you are using all defaults):

`data = random(DateRangeRandomizerTest.TestData.class);`